robusta/build.rs

```diff
@@ -31,9 +37,9 @@ fn main() {
 
     // Check for errors in `npm install`
     if !npm_install.status.success() {
-        panic!(
-            "`npm install` failed: {}",
-            String::from_utf8_lossy(&npm_install.stderr)
+        println!(
+            "cargo:warning=`npm install` failed: {}",
+            String::from_utf8_lossy(&npm_install.stderr).replace('\n', "\\n")
```

Comment:
```suggestion
            String::from_utf8_lossy(&npm_install.stderr).replace('\n', "\ncargo:warning="),
```
The error is pretty unreadable otherwise, although having every line prefixed with `warning: ` isn't ideal either.